src/pages/AppProfile.vue

```diff
@@ -96,6 +96,9 @@
               </div>
               <div class="flex column details">
                 <span class="body-2">Size</span>
+                <span class="body-3" v-if="organisationDetails.sizeErr">
```

Comment:
Any suggestions for the typography? Hierarchically this is most appropriate but it's too small to read...

Reply:
We could try making the text uppercase and increasing the letter spacing.

```css
text-transform: uppercase;
letter-spacing: .1em;
```